diff -ur guile-reader-0.6.3.orig/configure.ac guile-reader-0.6.3/configure.ac
--- guile-reader-0.6.3.orig/configure.ac	2020-07-27 00:39:30.000000000 +0200
+++ guile-reader-0.6.3/configure.ac	2021-06-05 09:26:24.339298050 +0200
@@ -107,7 +107,7 @@
 GUILE_PKG([3.0 2.2 2.0 1.8])
 GUILE_PROGS
 GUILE_FLAGS
-AC_PATH_PROG([guile_snarf], [guile-snarf], [not-found])
+AC_PATH_PROGS([guile_snarf], [guile-snarf-$GUILE_EFFECTIVE_VERSION guile-snarf], [not-found])
 if test "x$guile_snarf" = "xnot-found"; then
    AC_MSG_ERROR([`guile-snarf' not found.  Please install Guile 1.8.x or later.])
 fi
