--- guile-lib-0.2.5.1/configure.ac.old	2019-06-23 16:28:07.240273454 +0200
+++ guile-lib-0.2.5.1/configure.ac	2019-06-23 16:28:35.024761924 +0200
@@ -61,7 +61,7 @@
 dnl although no change has been made to any module, we bumped the
 dnl dependency version to 2.0.12 [or higher], because it has texinfo
 dnl patches so 'make check' pass.
-GUILE_PROGS([2.0.12])
+GUILE_PROGS
 GUILE_SITE_DIR
 GUILE_GLOBAL_SITE_DIR
 GUILE_SITE_CCACHE_DIR
--- guile-lib-0.2.5.1/configure.ac.old	2019-06-23 16:52:49.220232515 +0200
+++ guile-lib-0.2.5.1/configure.ac	2019-06-23 16:51:57.859334264 +0200
@@ -70,7 +70,7 @@
 dnl this should disappear
 dnl but for now we'll keep it, it's used 'en aval'
 HAVE_GUILE_COMPILE=1
-GUILE_COMPILE="guild compile"
+GUILE_COMPILE="$GUILD compile"
 AC_SUBST(GUILE_COMPILE)
 AM_CONDITIONAL(HAVE_GUILE_COMPILE, test "$HAVE_GUILE_COMPILE" = 1)
 
