--- guile-lib-0.2.6.1/configure.ac.old	2019-06-23 16:28:07.240273454 +0200
+++ guile-lib-0.2.6.1/configure.ac	2019-06-23 16:28:35.024761924 +0200
@@ -61,7 +61,7 @@
 dnl although no change has been made to any module, we bumped the
 dnl dependency version to 2.0.12 [or higher], because it has texinfo
 dnl patches so 'make check' pass.
-GUILE_PROGS([2.0.12])
+GUILE_PROGS
 GUILE_SITE_DIR
 GUILE_GLOBAL_SITE_DIR
 GUILE_SITE_CCACHE_DIR
--- guile-lib-0.2.6.1/am/guile.mk.old	2018-06-25 06:12:19.000000000 +0200
+++ guile-lib-0.2.6.1/am/guile.mk	2019-06-23 17:50:00.497178155 +0200
@@ -27,6 +27,7 @@
 
 moddir=@SITEDIR@
 godir=@SITECCACHEDIR@
+guild=@GUILD@
 
 
 GOBJECTS = $(SOURCES:%.scm=%.go)
@@ -47,4 +48,4 @@
 SUFFIXES = .scm .go
 .scm.go:
 	$(AM_V_GEN)$(top_builddir)/pre-inst-env \
-	guild compile $(GUILE_WARNINGS) -o "$@" "$<"
+	$(guild) compile $(GUILE_WARNINGS) -o "$@" "$<"
