diff -Nuar a/linedit.asd b/linedit.asd
--- a/linedit.asd	2014-07-26 14:24:03.109913304 +0200
+++ b/linedit.asd	2014-07-26 14:25:34.279912792 +0200
@@ -75,10 +75,13 @@
    (:file "matcher" :depends-on ("packages"))
 
    ;; Backend
-   (:file "backend" :depends-on ("utility-macros"))
-   (:uffi-c-source-file "terminal_glue")
+   ;(:file "backend" :depends-on ("utility-macros"))
+   ;(:uffi-c-source-file "terminal_glue")
+   (:file "uffi-loader" :depends-on ("packages"))
+   (:file "backend" :depends-on ("utility-macros" "uffi-loader"))
+		   
    (:file "terminal-translations" :depends-on ("packages"))
-   (:file "terminal" :depends-on ("terminal-translations" "backend" "terminal_glue"))
+   (:file "terminal" :depends-on ("terminal-translations" "backend" "uffi-loader"))
    (:file "smart-terminal" :depends-on ("terminal" "matcher"))
    (:file "dumb-terminal" :depends-on ("terminal"))
 
